react/features/base/tracks/functions.js

```diff
@@ -53,16 +53,21 @@ export function createLocalTracksF(
 
     const state = store.getState();
     const {
-        constraints,
         desktopSharingFrameRate,
         firefox_fake_device, // eslint-disable-line camelcase
         resolution
     } = state['features/base/config'];
-    const loadEffectsPromise = state['features/blur'].blurEnabled
-        ? getBlurEffect()
+    const constraints = options.constraints
+        ?? state['features/base/config'].constraints;
+
+    // Do not load blur effect if option for ignoring effects is present.
+    // This is needed when we are creating a video track for presenter mode.
+    const loadEffectsPromise = !options.ignoreEffects
+        && state['features/blur'].blurEnabled ? getBlurEffect()
             .then(blurEffect => [ blurEffect ])
             .catch(error => {
-                logger.error('Failed to obtain the blur effect instance with error: ', error);
+                logger.error('Failed to obtain the blur effect instance with'
```

Comment:
unnecessary change?

Reply:
Yes, it was 95 characters long but I reverted it now.